hack/run_test.sh

```diff
@@ -4,7 +4,7 @@ THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 export ARTIFACT_DIR=${ARTIFACT_DIR:="/tmp/gpu-test"}
 export KUBECONFIG=${KUBECONFIG:="$HOME/.kube/config"}
-export GPU_OPERATOR_MASTER_BUNDLE="registry.gitlab.com/nvidia/kubernetes/gpu-operator/staging/gpu-operator-bundle:master-latest"
+export GPU_OPERATOR_MASTER_BUNDLE="registry.gitlab.com/nvidia/kubernetes/gpu-operator/staging/gpu-operator:release-23.03-latest-ubi8"
```

Comment:
```suggestion
export GPU_OPERATOR_MASTER_BUNDLE="registry.gitlab.com/nvidia/kubernetes/gpu-operator/staging/gpu-operator-bundle:release-23.03-latest"
```

I couldn't find the image path you're suggesting, but I could find this one: registry.gitlab.com/nvidia/kubernetes/gpu-operator/staging/gpu-operator-bundle:release-23.03-latest [here](https://gitlab.com/nvidia/kubernetes/gpu-operator/container_registry/3429980?orderBy=NAME&sort=asc&search[]=release&search[]=).

Reply:
Noticed that as well... Ill ask them for the right image.

Reply:
Ill change to registry.gitlab.com/nvidia/kubernetes/gpu-operator/staging/gpu-operator-bundle:release-23.03-latest for a test. I'm still waiting for a response from NVIDIA for the right image to track. 

I'll hold the PR until then.

Reply:
Seems like this image is not the latest. It's still 23.3.0